src/Microsoft.Dnx.Runtime.Sources/Impl/EnvironmentNames.cs

```diff
@@ -16,12 +16,9 @@ internal static class EnvironmentNames
         public const string AppBase = "DNX_APPBASE";
```

Comment:
Why? I don't see any code that reads this variable... (besides the one I removed because we pass the appBase in the bootstrapper context) Also, don't we always pass the `--appbase` parameter from the script?